Assets/Scripts/Game/Questing/Quest.cs

```diff
@@ -178,10 +178,7 @@ public IMacroContextProvider ExternalMCP
         /// <summary>
         /// Gets world time of quest when started.
         /// </summary>
-        public DaggerfallDateTime QuestStartTime
-        {
-            get { return questStartTime; }
-        }
+        public DaggerfallDateTime QuestStartTime { get ; set ; }
```

Comment:
Ah, we should document these "coding standards" somewhere, this isn't the first time I missed a convention.

Reply:
I actually use auto-properties a lot, i think they are a great way to avoid additional syntax that increase the length of a file and doesn't really provide more informations when you read it.

Reply:
I have no problem with auto properties either. I think they were not used historically due to c# version or just a lack of c# knowledge in devs?

Although usually there's no space between get/set and the ;  
(sorry, nitpicking there)

Reply:
Or maybe it was because Unity didn't support C# 3.0 at first? Don't know, but nevermind, this is a very personal point of view on my side, and unrelated with this PR. Since you all agree with using this feature, no problem there :)